jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -1488,50 +1515,51 @@ YUI.add('juju-env-go', function(Y) {
       @return {undefined} Sends a message to the server only.
     */
     _add_unit: function(service, numUnits, toMachine, callback) {
```

Comment:
is the description for toMachine no longer valid based on the code in `parsePlacement`? Does the syntax of the placement need to change for the app code which is calling `add_unit` ?